### ISSUE TITLE
get_per_capita_household_size() uses outdated LGA definitions

### ISSUE THREAD

**Issue by goldingn** [njtierney/conmat#50]
`abs_household_lga` uses data with an older set of LGAs than `abs_pop_age_lga_2020`, so the code to adjust household sizes doesn't work for all LGAs.

We'll need to use a correspondence to line them up. I'll see what I can hack together for this one off code I'm writing and paste here.

**Comment by aarathybabu97** [comment]
`
mutate(
  lga = case_when(
    lga == "Botany Bay (C)" ~ "Bayside (A)",
    lga == "Gundagai (A)" ~ "Cootamundra-Gundagai Regional (A)",
    lga == "Nambucca (A)" ~ "Nambucca Valley (A)",
    lga == "Western Plains Regional (A)" ~ "Dubbo Regional (A)",
    lga == "Mallala (DC)" ~ "Adelaide Plains (DC)",
    lga == "Orroroo/Carrieton (DC)" ~ "Orroroo-Carrieton (DC)",
    lga == "Break O'Day (M)" ~ "Break O`Day (M)",
    lga == "Glamorgan/Spring Bay (M)" ~ "Glamorgan-Spring Bay (M)",
    lga == "Waratah/Wynyard (M)" ~ "Waratah-Wynyard (M)",
    lga == "Kalamunda (S)" ~ "Kalamunda (C)",
    lga == "Kalgoorlie/Boulder (C)" ~ "Kalgoorlie-Boulder (C)",
    TRUE ~ lga
  )
)
`

**Comment by aarathybabu97** [comment]
```r
mutate(
  lga = case_when(
    lga == "Botany Bay (C)" ~ "Bayside (A)",
    lga == "Gundagai (A)" ~ "Cootamundra-Gundagai Regional (A)",
    lga == "Nambucca (A)" ~ "Nambucca Valley (A)",
    lga == "Western Plains Regional (A)" ~ "Dubbo Regional (A)",
    lga == "Mallala (DC)" ~ "Adelaide Plains (DC)",
    lga == "Orroroo/Carrieton (DC)" ~ "Orroroo-Carrieton (DC)",
    lga == "Break O'Day (M)" ~ "Break O`Day (M)",
    lga == "Glamorgan/Spring Bay (M)" ~ "Glamorgan-Spring Bay (M)",
    lga == "Waratah/Wynyard (M)" ~ "Waratah-Wynyard (M)",
    lga == "Kalamunda (S)" ~ "Kalamunda (C)",
    lga == "Kalgoorlie/Boulder (C)" ~ "Kalgoorlie-Boulder (C)",
    TRUE ~ lga
  )
)
```

**Comment by njtierney** [comment]
Thanks, @aarathybabu97 !

@goldingn - did this end up being resolved in another way in some of the other analyses, or would we be good to include something like the changes Aarathy has provided above?

**Comment by goldingn** [comment]
Yeah, awesome!

Worth writing a test to make sure all the LGAs work? That way we will know
something is up if we change the LGA data and it all breaks.

**Comment by njtierney** [comment]
I think this has been addressed in #61 , @goldingn

**Comment by aarathybabu97** [comment]
Reopening this issue as the `abs_household_lga` wasn't updated before in #61. Updated in PR #68 with appropriate tests.
LGAs that couldn't be renamed are as follows
```
library(dplyr)
library(conmat)
missing_lgas <- anti_join(abs_household_lga,abs_pop_age_lga_2020,by="lga")
unique(missing_lgas$lga)
#> [1] "Migratory - Offshore - Shipping (NSW)" 
#> [2] "Migratory - Offshore - Shipping (Vic.)"
#> [3] "Migratory - Offshore - Shipping (Qld)" 
#> [4] "Migratory - Offshore - Shipping (SA)"  
#> [5] "Migratory - Offshore - Shipping (WA)"  
#> [6] "Migratory - Offshore - Shipping (Tas.)"
#> [7] "Migratory - Offshore - Shipping (NT)"  
#> [8] "Migratory - Offshore - Shipping (ACT)"
Created on 2022-04-22 by the reprex package (v2.0.1)
```
